types/cavy/cavy-tests.tsx

```diff
@@ -48,7 +48,7 @@ class SampleComponent extends React.Component<Props> {
           text='text'
         />
 
-        <WrappedText ref={generateTestHook('WrappedText')}>
+        <WrappedText accessibilityRole='button' ref={generateTestHook('WrappedText')}>
```

Comment:
`WrappedText` previously didn't allow any props that are possible on `Text` except `children`. But `children` were only accepted since `@types/react` implicitly adds `children` to every component. However, implicit children will be removed when React 18 is released (see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/46691). Fortunately, removing implicit children will not be breaking if this change (which improves other types as well) is merged.